src/sample/downloadController.java

```diff
@@ -37,38 +46,56 @@ public static void inputID(String text){
          stuID=text;
   }
   @FXML
-  public void handleButtonAction(ActionEvent event) throws IOException {
+  public void handleButtonAction(ActionEvent event) throws FileNotFoundException,IOException {
     final DirectoryChooser dirchooser = new DirectoryChooser();
 
     Stage stage = (Stage) anchorid.getScene().getWindow();
     File file = dirchooser.showDialog(stage);
 
-    if (file != null) {
-      List list1 = new List();
-      // input the text file for the course list
+     if (file != null) {
+
+      System.out.println("Path: " + file.getAbsolutePath());
+      textfield.setText(file.getAbsolutePath());
+
+      //Image
+      ImageData data= ImageDataFactory.create("C:/Users/user/student-course-registration-system/src/sample/images/usm-ringlogo.png");
+      Image image1 = new Image(data);
+      image1.scaleToFit(140f,120f);
+      image1.setHorizontalAlignment(HorizontalAlignment.CENTER);
+      
+      //Table
+      float TableColWidth[] = {50f,300f}; 
+      Table courseTable = new Table(TableColWidth);
+
+      courseTable.addCell("No.");
+      courseTable.addCell("Course Taken");
+
+      //Print course list in table
       try {
         File fileObj = new File("data/Student Registered Course/"+stuID+".txt");
         Scanner fileReader = new Scanner(fileObj);
+        int count=1;
         while (fileReader.hasNextLine()) {
-          list1.add(fileReader.nextLine());
+          courseTable.addCell(Integer.toString(count));
+          courseTable.addCell(fileReader.nextLine());
+          count++;
         }
         fileReader.close();
       } catch (FileNotFoundException e) {
         System.out.println("An error occurred.");
         e.printStackTrace();
       }
-
-      System.out.println("Path: " + file.getAbsolutePath());
-      textfield.setText(file.getAbsolutePath());
-
       PdfWriter pdfWriter = new PdfWriter(file.getAbsolutePath() + "/CourseList.pdf");
 
       PdfDocument pdfDocument = new PdfDocument(pdfWriter);
       pdfDocument.addNewPage();
 
       Document document = new Document(pdfDocument);
-      document.add(list1);
-
+      document.add(image1);
+      document.add(new Paragraph("Universiti Sains Malaysia").setTextAlignment(TextAlignment.CENTER).setBold().setFontSize(30f));
+      document.add(new Paragraph("Student ID:   "+stuID));
```

Comment:
Can add name or not?

Reply:
If the user didn't put the name, just display "User didn't update the name" smthg like that